channels-and-providers/sms/46elks.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: '46elks'
+description: 'Learn how to use the 46elks provider to send sms notifications using Novu'
+---
+
+[46elks](https://46elks.com/) API let's you intigrate SMS, MMS and phone calls into your applications with just a few lines of code.
+
+# Getting Started
+
+To use the 46elks provider in the sms channel, you will need to create a 46elks account and add your `API Username` & `API Password` to the 46elks integration on the Novu platform.
+
+# Creating the 46elks integration with Novu
+
+- Visit the [Integrations](https://web.novu.co/integrations) page on Novu.
+- Select 46elks service
```

Comment:
The second step is to click on "Add a provider" before selecting 46elks provider. So please add that.